plugins/pubIds/doi/locale/es_ES/locale.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/es_ES/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings.
+  -->
+
+<locale name="es_ES" full_name="Español (España)">
+	<message key="plugins.pubIds.doi.displayName">DOI</message>
+	<message key="plugins.pubIds.doi.description">Este complemento permite la asignación del Identificador de Objeto Digital a publicaciones, artículos y galeradas en OJS.</message>
+	<message key="plugins.pubIds.doi.readerDisplayName">DOI</message>
+	<message key="plugins.pubIds.doi.manager.settings.description">Por favor, configure el complemento de DOI para poder administrar y usar DOIs en OJS:</message>
```

Comment:
From the translation style book:
- Avoid "Please" at the begining. Makes reading too slow.
- Plugin need to be translated as "módulo".

<message key="plugins.pubIds.doi.manager.settings.description">Configure el módulo DOI para poder administrar y usar DOIs en OJS:</message>